aws/repositories/appointments_repository.go

```diff
@@ -60,13 +86,13 @@ func (r AppointmentsRepository) CreateAppointment(req events.APIGatewayProxyRequ
 	return nil, nil
 }
 
-func (r AppointmentsRepository) createAppointment(counselor *counselor.Counselor, user *user.User, duration time.Duration, time time.Time) error {
+func (r AppointmentsRepository) createAppointment(counselor_id string, user_id string, duration time.Duration, time time.Time) error {
```

Comment:
Same thing, we should try to create appointment passing counselor and user ... ideally, we would want to verify counselor and user id before attempting to create an appoinemnt

Reply:
i dont follow could you pls elaborate , we need to save id and not the entire user or counselor object , will ad the part to do validity check

Reply:
Mentioned the reasoning in the other comment

Reply:
https://github.com/sal-org/sal-backend/pull/4/files#r534674037